data/roles.js

```diff
@@ -714,13 +714,38 @@ const roleData = {
                 "Wins if a certain number of missions are successful."
             ]
         },
+        "Merlin": {
+            alignment: "Resistance",
+            description: [
+                "Kowns the alignment of all spies.",
```

Comment:
Knows everyone who is aligned with the spies?

Reply:
yes